client/my-sites/site-settings/cloudflare.js

```diff
@@ -43,7 +43,7 @@ const Cloudflare = () => {
 								<p className="site-settings__cloudflare-title">
 									{ translate( 'Jetpack Site Accelerator' ) }
 								</p>
-								<p>{ translate( 'Comes built-in with WordPress.com Business plans.' ) }</p>
+								<p>{ translate( 'Comes built-in with WordPress.com Pro plan.' ) }</p>
```

Comment:
Should probably have a `the` before WordPress.com Pro

Reply:
Fixed by https://github.com/Automattic/wp-calypso/pull/61595/commits/e97f4cdc8360039da0ffde3243130689c5869ca3